### ISSUE TITLE
Display issue of SharePoint tags on card view of PnP Modern Search 

### ISSUE THREAD

**Issue by FelloSimon** [microsoft-search/pnp-modern-search#3797]
Hello,

I want to display the tags on the card views as well but by doing so (through "manage card fields", the entire internal name stays visible.

![image](https://github.com/microsoft-search/pnp-modern-search/assets/172600347/d64bf199-5a5e-4fa9-9349-491c45139920)

Does someone has any idea what handle bar or setting I have to configure to only display the part after "|"?(aka the tag name)

Tags on the first card should be "Marketing" and "Policy". Tags of the second card should be "Projects" and "Policy". Tags of the third card should be "Projects" and "Policy".

Thanks

**Comment by wobba** [comment]
See "Enable localization" at https://microsoft-search.github.io/pnp-modern-search/usage/search-results/data-sources/#source-configuration

**Comment by kasperbolarsen** [comment]
If you map a RefinableString to the crawled properties you have a choice,  either map to the *taxid* one, and get the entire property,  or to the one without the *taxid*, and in that case you get the term label value in the RefinableString. 
If you have multi language terms and need to show those, you cannot use the approach above.

**Comment by sympmarc** [comment]
The important part of the instructions @wobba pointed you to, @FelloSimon, is this section:

_To use translated values in your template, you must use the 'Auto + <property_name>' property format instead of the original property name. For instance, to use translated values of the 'owstaxidmetadataalltagsinfo' property, you must use the 'Autoowstaxidmetadataalltagsinfo' auto created property._

You do this by customizing layout slots (screenshot 1) and adding the `Auto` the prefix (screenshot 2)

Does that solve your issue?

![image](https://github.com/microsoft-search/pnp-modern-search/assets/1295627/ee1b77b7-f352-494c-a011-e14b17b6039f)


![image](https://github.com/microsoft-search/pnp-modern-search/assets/1295627/bc1e892a-6d0d-48fb-826e-fa7ecb1f9c7e)

**Comment by FelloSimon** [comment]
> See details about "Enable localization" at https://microsoft-search.github.io/pnp-modern-search/usage/search-results/data-sources/#source-configuration
I already switced on "Enable Localization".

**Comment by FelloSimon** [comment]
> The important part of the instructions @wobba pointed you to, @FelloSimon, is this section:
> 
> _To use translated values in your template, you must use the 'Auto + <property_name>' property format instead of the original property name. For instance, to use translated values of the 'owstaxidmetadataalltagsinfo' property, you must use the 'Autoowstaxidmetadataalltagsinfo' auto created property._
> 
> You do this by customizing layout slots (screenshot 1) and adding the `Auto` the prefix (screenshot 2)
> 
> Does that solve your issue?
> 
> ![image](https://private-user-images.githubusercontent.com/1295627/340374916-ee1b77b7-f352-494c-a011-e14b17b6039f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg3MDkyNjIsIm5iZiI6MTcxODcwODk2MiwicGF0aCI6Ii8xMjk1NjI3LzM0MDM3NDkxNi1lZTFiNzdiNy1mMzUyLTQ5NGMtYTAxMS1lMTRiMTdiNjAzOWYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYxOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MThUMTEwOTIyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NTk2Y2MxY2ZjZDY4NTVkZjJkODg5MjMyZmY3NTQ1NzU3MTkzMzU3YTM5MDIxMjhmOTc4MzVjNDYzNWY3YTRlYSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.ukV-ITYyVfZO-qmEQ7J8NdAeu9tlougNvGIwYNnBcTc)
> 
> ![image](https://private-user-images.githubusercontent.com/1295627/340374882-bc1e892a-6d0d-48fb-826e-fa7ecb1f9c7e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg3MDkyNjIsIm5iZiI6MTcxODcwODk2MiwicGF0aCI6Ii8xMjk1NjI3LzM0MDM3NDg4Mi1iYzFlODkyYS02ZDBkLTQ4ZmItODI2ZS1mYTdlY2IxZjljN2UucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYxOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MThUMTEwOTIyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MWYzZjQzYjg5Y2JiMzRmZmQyZTg0YzFmYzk5YjlkMTNlYWNlY2U1ZTA0ODcwMTEyM2IyNGQxM2Y4NjkxYTkxNyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.kRBxWsV84uXz_8JdMTm_81KY-oq6qCDDf1EdCP9Rjyw)

Yes this worked out! Thanks a lot. The only problem I now have is that the tags aren't shown individually but only as one single string
<img width="457" alt="image" src="https://github.com/microsoft-search/pnp-modern-search/assets/172600347/c2094581-2776-4285-93c2-148c3677536d">
